lib/Config/Populator.php

```diff
@@ -688,6 +702,50 @@ private function updateSentUnsubscribeLinksToInstantUnsubscribeLinks() {
     ));
   }
 
+  private function addPlacementStatusToForms() {
+    if (version_compare($this->settings->get('db_version', '3.48.1'), '3.48.1', '>')) {
```

Comment:
I think this should be `$this->settings->get('db_version', '3.48.2')` so that it doesn't run for new installs and also next release will be minor so maybe even '3.49.0'